joschahenningsen/TUM-Live
# Get thumbnails for livestreams

### Description
Implements https://github.com/joschahenningsen/TUM-Live/issues/957

A cron job is created that fetches thumbnails every minute and writes them to `tmp` as a jpeg with the stream id as the file name. By using an admin token, workers should have access to the HLS stream.
I annotated the worker Docker files with `amd64` so the `docker-compose` setup works on ARM macs (tesseract has issues with ARM) such that this PR was easier to test for me.

### Steps for Testing
1. Use local `docker-compose` setup.
2. Go to the maintenance page in the settings and run `fetchLivePreviews`.
3. Observe that the log has no errors and the file is created.
